### DISCUSSION TITLE
Error during "CalculatePerformanceStats"

### DISCUSSION THREAD

**Discussion by HugoPere** [esperecyan/VRMConverterForVRChat#18] [Q&A] [answered]
Hi, i tried to install the package, and it went well, but as the title says, a "CalculatePerformanceStats" error happens as you can see in the screenshot. 
![image](https://user-images.githubusercontent.com/50475071/66002997-5635fd80-e47b-11e9-9526-65bb46997d84.png) 

 Is this an installation problem? Because the "clone to vrchat" option doesn't appear, and I think that is because of this. 

If you don't understand english, i'll try to use google translate. 

-----------

Google翻訳、エラーでごめんなさい

こんにちは、パッケージをインストールしようとしましたが、うまくいきましたが、タイトルにあるように、スクリーンショットでわかるように「CalculatePerformanceStats」エラーが発生します。
![image](https://user-images.githubusercontent.com/50475071/66002997-5635fd80-e47b-11e9-9526-65bb46997d84.png) 

これはインストールの問題ですか？ 「vrchatにクローンを作成する」オプションが表示されないため、これが原因だと思います。

**Comment by esperecyan** [comment, marked as answer]
VRM Converter for VRChat v16.0.0 is not compatible with VRChat SDK 2019.08.23.13.45 or older.
https://github.com/esperecyan/VRMConverterForVRChat/releases/tag/v16.0.0

Update VRChat SDK to the latest version (2019.09.18.12.05).
https://docs.vrchat.com/docs/updating-the-sdk
